lib/projects.ex

```diff
@@ -104,11 +105,68 @@ defmodule GlassFactoryApi.Projects do
       iex> GlassFactoryApi.Projects.get_project!("999")
       ** (ArgumentError) Can't find a project with 999
   """
+  @spec get_project!(String.t(), map()) :: Project.t()
   def get_project!(project_id, config \\ %{}) do
     with {:ok, project} <- get_project(project_id, config) do
       project
     else
       {:error, error} -> raise error
     end
   end
+
+  @doc """
+  Returns the list of members of a given project.
+
+  If the request returns successfully then `{:ok, [%ProjectMember{}]}` will be returned, but if something went wrong `:error` is returned along with a message.
+
+  ## Examples
+
+      iex> GlassFactoryApi.Projects.list_members("123")
+      {:ok, [ %ProjectMember{
+            id: "123456",
+            user_id: "123",
+            project_id: "456"
+          }
+        ]
+      }
+
+      iex> GlassFactoryApi.Projects.list_members("999")
+      [:error, "Can't find a project with ID 999"]
+  """
+  @spec list_members(String.t(), map()) :: {atom(), ProjectMember.t() | String.t()}
+  def list_members(project_id, config \\ %{}) do
+    case ApiClient.get("projects/#{project_id}/members", config) do
+      {:ok, %{status: 200, body: body}} -> {:ok, Enum.map(body, &ProjectMember.to_struct(&1))}
+      {:ok, %{status: 404}} -> {:error, "Can't find a project with ID #{project_id}"}
```

Comment:
We will probably want to make this error more _matchable_, as in `{:error, {:not_found, "Can't find a project with ID #{project_id}"}}` or something similar.

Here's some food for thought https://michal.muskala.eu/2017/02/10/error-handling-in-elixir-libraries.html

Will the API itself return some body text when it returns 404? If that's the case, we can also wrap the original text.

Reply:
The API doesn't return a body with 404.

`defexception` is TIL